src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -459,7 +459,8 @@ private string UFormatDateString(DateTime dateTime)
                             break;
 
                         case 'V':
-                            sb.Append((dateTime.DayOfYear / 7) + 1);
+                            GregorianCalendar calendar = new GregorianCalendar();
```

Comment:
I don't believe using the CurrentCulture is appropriate in this case, because ISO 8601 is Gregorian calendar based and CurrentCulture could be set to a different calendar.

In any case further tests based on the table at https://en.wikipedia.org/wiki/ISO_week_date#Relation_with_the_Gregorian_calendar shows that my initial stab at this was incorrect, because the GetWeekOfYear does not produce the  ISO 8601 output for all cases. I'll take a bit of a harder stare at this.